tests/hpxml_translator_test.rb

```diff
@@ -42,7 +42,7 @@ def test_simulations
 
     xmls = []
     test_dirs.each do |test_dir|
-      Dir["#{test_dir}/base*.xml"].sort.each do |xml|
+      Dir["#{test_dir}/base-foundation*.xml"].sort.each do |xml|
```

Comment:
Is this intended?

Reply:
Oops. 🤦‍♂ No, I was just testing those files to make sure I handled the merge correctly. Good catch, please fix!